### PR TITLE
Add missing tool dependencies for package_fontconfig_2_11_1

### PR DESCRIPTION
**Pull request by methylome**
package_zlib_1_2_8 and package_libpng_1_6_7 were added for tool dependencies.

### CODE REVIEW

File: packages/package_fontconfig_2_11_1/tool_dependencies.xml [hunk 1]
```diff
@@ -1,46 +1,56 @@
 <?xml version="1.0"?>
 <tool_dependency>
     <package name="freetype" version="2.5.2">
-        <repository name="package_freetype_2_5_2" owner="iuc" prior_installation_required="True" />
+        <repository changeset_revision="3b09e7916896" name="package_freetype_2_5_2" owner="iuc" prior_installation_required="True" toolshed="https://toolshed.g2.bx.psu.edu" />
```
Comment (bgruening) [inline packages/package_fontconfig_2_11_1/tool_dependencies.xml]: Please remove changeset revision and toolshed. This is automatically filled during tool upload.

File: packages/package_fontconfig_2_11_1/tool_dependencies.xml [hunk 2]
```diff
@@ -1,46 +1,56 @@
...
     </package>
     <package name="libxml2" version="2.9.1">
-        <repository name="package_libxml2_2_9_1" owner="iuc" prior_installation_required="True" />
+        <repository changeset_revision="e5475acebd6e" name="package_libxml2_2_9_1" owner="iuc" prior_installation_required="True" toolshed="https://toolshed.g2.bx.psu.edu" />
```
Comment (bgruening) [inline packages/package_fontconfig_2_11_1/tool_dependencies.xml]: Changeset_revision and toolshed removal everywhere

File: packages/package_fontconfig_2_11_1/tool_dependencies.xml [hunk 3]
```diff
@@ -1,46 +1,56 @@
...
+                        <repository changeset_revision="3b09e7916896" name="package_freetype_2_5_2" owner="iuc" prior_installation_required="True" toolshed="https://toolshed.g2.bx.psu.edu">
                             <package name="freetype" version="2.5.2" />
                         </repository>
-                        <repository name="package_libxml2_2_9_1" owner="iuc" prior_installation_required="True">
+                        <repository changeset_revision="e5475acebd6e" name="package_libxml2_2_9_1" owner="iuc" prior_installation_required="True" toolshed="https://toolshed.g2.bx.psu.edu">
                             <package name="libxml2" version="2.9.1" />
                         </repository>
                     </action>
                     <action type="shell_command">
-                        <![CDATA[
```
Comment (bgruening) [inline packages/package_fontconfig_2_11_1/tool_dependencies.xml]: Please keep cdata ...

**Comment by methylome** [comment]
I've modified as you requested and the tool_shed attribute. Since I don't know how to reflect the modification to this pull request, I've created a new pull request.

**Comment by martenson** [comment]
@methylome if you push to your `epigenome:feature/package_fontconfig_2_11_1` branch (from which you issued this) this PR will update automatically

**Comment by methylome** [comment]
I got it. Thanks.

**Pull request closed without merging**